docs/source/intro/overview.rst

```diff
@@ -212,6 +212,42 @@ Agent can run both from container and from a local machine.
 
         python3 -m core.run -ch telegram
 
+    or via HTTP api client:
+
+    .. code:: bash
+
+        python3 -m core.run -ch http_client [-p 8888]
+
+    HTTP client accepts POST requests with application/json content-type.
```

Comment:
Here are two modes of how Agent can run: `Container` and `Local machine`. We need to specify that `http_client` is a valid option for both modes, not only `Local machine`.